frontend/src/fileSender.js

```diff
@@ -51,6 +51,9 @@ class FileSender extends EventEmitter {
         reader.onload = function(event) {
           resolve(new Uint8Array(this.result));
         };
+        reader.onerror = function(event) {
+          reject(event.explicitOriginalTarget.error.name);
```

Comment:
I think that only sending the error to Raven here is sufficient? This will eliminate any duplicates, since errors from the `reader.onerror` triggers (https://github.com/mozilla/something-awesome/pull/168#discussion-diff-126022292R54) are always caught here.